frida_tools/tracer.py

```diff
@@ -14,6 +14,7 @@
 
 from frida_tools.model import Module, Function, ModuleFunction, ObjCMethod
 
+isQuiet = False
```

Comment:
This should go in `_initialize` and be assigned from the `options` argument, and PEP-8 style i.e. `is_quiet`.